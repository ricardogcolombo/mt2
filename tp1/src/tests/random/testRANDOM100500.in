100 500
1 1 108 31 102
1 50 65 63 128
1 4 1 88 120
1 9 56 87 55
1 46 126 57 67
1 8 65 28 19
1 57 93 9 115
1 80 1 17 73
1 23 88 89 130
1 94 46 47 71
1 13 86 61 103
1 77 76 88 13
1 86 44 77 121
1 88 35 34 15
1 5 100 88 58
1 5 38 43 22
1 43 70 41 50
1 70 90 58 57
1 42 20 36 101
1 54 103 63 101
1 51 130 97 31
1 93 15 55 49
1 77 25 56 66
1 62 95 71 8
1 62 6 45 116
1 66 98 86 20
1 43 103 65 60
1 31 37 30 42
1 52 22 23 48
1 79 24 65 93
1 52 104 42 129
1 99 69 47 62
1 50 16 38 24
1 63 91 21 89
1 99 76 90 101
1 11 111 98 122
1 78 83 77 72
1 6 83 34 95
1 79 130 95 54
1 32 44 30 50
1 31 54 27 97
1 33 96 72 45
1 39 60 20 71
1 96 84 30 20
1 71 53 68 115
1 28 108 12 98
1 68 50 67 129
1 51 123 8 104
1 42 45 2 48
1 31 15 34 5
1 88 72 48 54
1 99 25 4 85
1 19 122 25 84
1 33 40 32 50
1 76 57 40 104
1 36 97 85 20
1 96 116 91 112
1 44 3 96 12
1 94 30 53 77
1 6 129 48 17
1 76 37 14 20
1 71 47 35 122
1 97 71 38 112
1 56 10 100 128
1 70 48 64 113
1 42 59 16 46
1 87 2 84 32
1 4 28 61 8
1 27 68 57 4
1 74 57 76 40
1 5 31 36 5
1 14 125 17 35
1 37 127 46 63
1 80 67 59 111
1 23 15 50 65
1 31 43 98 40
1 34 32 25 83
1 97 35 14 25
1 25 111 87 119
1 6 50 83 43
1 17 77 67 112
1 83 51 23 11
1 64 77 65 103
1 56 105 39 119
1 62 72 4 54
1 76 64 79 4
1 44 111 45 1
1 48 75 11 13
1 100 52 97 64
1 14 126 8 22
1 8 79 15 28
1 14 85 16 12
1 6 129 44 84
1 18 1 5 35
1 80 118 56 73
1 80 95 11 12
1 69 7 35 1
1 13 103 35 24
1 82 70 36 75
1 13 88 49 12
1 57 59 71 44
1 46 23 76 115
1 14 34 69 48
1 89 19 33 94
1 71 127 9 57
1 35 71 54 119
1 17 68 89 39
1 70 52 82 108
1 1 89 49 129
1 44 122 48 107
1 48 28 54 68
1 51 6 28 18
1 54 96 17 20
1 93 57 15 130
1 55 15 86 79
1 9 55 73 11
1 61 83 35 34
1 88 83 70 117
1 25 112 83 95
1 35 38 39 58
1 27 39 17 36
1 24 22 95 85
1 21 121 94 115
1 70 84 87 114
1 73 64 29 35
1 21 90 90 127
1 7 35 18 19
1 79 35 99 83
1 10 90 86 77
1 54 37 58 19
1 8 6 93 68
1 9 112 38 80
1 80 42 2 15
1 30 75 95 43
1 90 17 77 46
1 40 45 25 67
1 46 110 42 1
1 87 21 19 126
1 70 54 91 46
1 63 58 25 79
1 100 15 84 73
1 35 100 17 55
1 31 60 26 110
1 17 49 7 11
1 89 102 78 101
1 85 23 20 77
1 90 83 26 74
1 6 82 80 88
1 29 16 8 70
1 42 32 93 27
1 12 102 56 77
1 25 14 20 70
1 72 57 96 94
1 39 102 60 112
1 55 127 51 77
1 97 95 47 40
1 13 57 92 96
1 9 104 72 130
1 92 80 12 89
1 99 41 100 50
1 16 94 91 98
1 87 89 3 94
1 65 129 33 102
1 48 95 63 120
1 51 116 63 23
1 56 89 43 85
1 82 45 40 92
1 67 87 20 35
1 69 119 31 74
1 53 30 98 122
1 96 94 93 42
1 17 90 15 10
1 61 96 48 98
1 13 73 64 56
1 29 69 36 87
1 66 85 2 52
1 76 40 74 72
1 82 49 93 120
1 81 45 90 31
1 55 5 74 8
1 83 99 77 117
1 25 10 2 32
1 75 62 96 41
1 90 49 92 43
1 55 12 82 71
1 21 126 62 8
1 47 96 9 10
1 31 55 50 107
1 15 63 96 119
1 20 18 14 30
1 57 35 43 118
1 53 74 26 70
1 77 27 64 25
1 44 25 28 2
1 92 93 31 40
1 21 85 35 90
1 17 81 98 91
1 89 65 98 40
1 96 6 92 87
1 24 105 83 59
1 24 126 79 71
1 31 107 27 52
1 82 111 68 127
1 36 55 37 73
1 21 43 69 117
1 11 38 37 3
1 91 8 71 34
1 16 4 20 1
1 73 23 49 110
1 31 81 27 37
1 36 107 11 68
1 34 75 20 46
1 16 19 60 40
1 91 53 13 45
1 19 106 73 87
1 60 3 4 101
1 77 98 85 125
1 17 58 36 96
1 68 36 84 66
1 24 66 22 126
1 58 73 16 63
1 33 63 65 110
1 8 90 64 66
1 68 37 13 81
1 39 26 59 98
1 91 30 53 26
1 13 21 70 129
1 54 35 50 111
1 63 101 3 118
1 79 15 59 30
1 90 59 16 13
1 80 118 24 8
1 65 97 62 13
1 44 90 18 94
1 56 17 31 80
1 96 117 57 2
1 66 76 93 89
1 67 103 64 122
1 100 24 54 111
1 63 15 21 87
1 89 65 92 41
1 8 33 85 43
1 3 71 45 20
1 38 27 36 74
1 31 21 85 83
1 24 84 74 99
1 60 20 54 46
1 83 40 34 4
1 77 27 55 9
1 28 101 7 24
1 71 71 99 43
1 79 111 70 85
1 57 35 66 78
1 92 7 69 1
1 27 82 51 83
1 21 39 13 14
1 28 55 21 10
1 64 124 58 1
1 9 45 23 118
1 6 44 63 57
1 67 24 26 128
1 10 47 77 75
1 98 2 69 21
1 51 46 42 124
1 36 67 57 60
1 20 71 91 42
1 78 43 53 17
1 43 79 33 3
1 67 100 64 27
1 97 23 5 82
1 37 64 89 70
1 37 88 71 96
1 11 29 80 54
1 97 10 83 26
1 16 82 79 4
1 72 21 63 112
1 13 19 17 99
1 8 68 75 119
1 66 66 85 34
1 62 110 80 40
1 33 23 36 82
1 67 96 66 23
1 19 37 22 72
1 69 84 14 20
1 6 118 40 52
1 56 81 27 42
1 100 39 63 54
1 92 12 80 109
1 50 48 66 62
1 35 1 60 109
1 50 63 20 46
1 26 122 64 16
1 78 91 76 30
1 26 101 45 82
1 29 64 13 109
1 48 58 43 117
1 9 35 26 66
1 65 33 70 125
1 45 130 21 86
1 6 18 14 72
1 11 53 55 67
1 79 44 9 58
1 89 126 31 44
1 2 41 57 13
1 89 83 9 93
1 96 84 92 21
1 63 57 78 4
1 33 119 30 31
1 19 112 57 90
1 72 1 95 130
1 6 71 45 0
1 7 0 43 101
1 73 10 62 65
1 29 100 38 48
1 59 105 47 39
1 24 104 13 88
1 68 118 16 25
1 84 66 39 110
1 72 45 46 16
1 8 81 59 41
1 71 40 40 119
1 57 29 18 24
1 29 67 54 66
1 37 45 64 110
1 88 90 51 86
1 62 110 20 107
1 46 6 93 16
1 47 71 42 76
1 47 74 19 65
1 31 38 2 90
1 65 116 92 97
1 58 64 83 46
1 93 53 88 71
1 24 54 65 70
1 2 27 19 3
1 95 94 49 102
1 64 82 50 34
1 65 67 59 1
1 18 27 32 102
1 40 39 44 7
1 27 63 64 31
1 20 61 96 48
1 2 31 61 67
1 66 10 17 73
1 76 84 23 128
1 99 120 81 128
1 33 37 47 76
1 10 49 93 48
1 51 74 72 29
1 76 4 77 122
1 41 21 59 32
1 24 127 41 95
1 27 73 69 71
1 77 31 90 74
1 12 34 16 48
1 65 66 24 120
1 10 25 81 112
1 14 123 45 95
1 46 102 96 106
1 31 112 8 84
1 43 130 49 126
1 5 34 23 10
1 4 82 53 21
1 34 33 53 89
1 89 130 60 85
1 75 28 89 15
1 69 76 47 56
1 18 130 57 37
1 30 31 39 70
1 28 121 54 96
1 7 34 87 71
1 47 18 3 83
1 14 114 37 122
1 45 42 23 130
1 17 19 19 24
1 19 81 53 40
1 77 45 40 119
1 77 68 39 107
1 26 79 13 13
1 1 52 79 43
1 21 107 55 66
1 19 2 97 67
1 45 57 33 46
1 2 101 89 130
1 90 24 14 14
1 94 45 40 21
1 58 6 55 103
1 74 25 21 73
1 30 26 22 49
1 36 44 9 35
1 59 24 83 42
1 99 76 20 88
1 16 105 39 129
1 50 95 96 18
1 94 24 33 121
1 72 78 34 102
1 29 126 80 100
1 19 60 1 36
1 44 90 48 62
1 43 116 19 47
1 43 0 25 22
1 34 88 83 57
1 27 55 62 12
1 27 122 17 20
1 3 105 99 84
1 38 104 33 49
1 79 96 92 39
1 32 91 47 112
1 17 16 20 40
1 61 34 64 9
1 12 40 76 18
1 26 77 72 121
1 17 68 15 24
1 25 31 87 4
1 88 35 86 107
1 40 0 30 85
1 53 72 5 93
1 23 66 50 27
1 70 55 5 13
1 54 37 14 102
1 100 87 8 94
1 7 127 89 94
1 81 126 68 100
1 99 107 58 38
1 82 72 56 68
1 92 98 54 11
1 39 120 23 79
1 11 31 51 119
1 73 13 33 21
1 77 111 56 20
1 48 50 20 62
1 57 69 36 104
1 22 30 82 16
1 11 67 47 106
1 17 42 61 79
1 25 18 82 125
1 98 89 31 112
1 66 125 81 16
1 47 76 27 46
1 28 108 39 125
1 99 11 80 101
1 24 11 98 74
1 56 32 2 118
1 16 46 43 93
1 60 46 84 108
1 95 76 18 28
1 40 95 95 47
1 61 63 19 28
1 77 123 25 65
1 17 42 63 3
1 14 95 47 89
1 71 106 17 119
1 71 58 3 75
1 34 84 80 77
1 41 48 5 43
1 23 111 18 77
1 44 108 50 27
1 38 85 96 6
1 64 115 23 116
1 25 35 100 69
1 43 35 67 98
1 71 95 42 99
1 86 81 24 100
1 52 64 37 20
1 22 43 50 31
1 45 70 40 40
1 29 61 23 66
1 73 56 83 106
1 1 92 52 78
1 88 111 82 17
1 3 126 25 14
1 45 54 98 126
1 95 32 36 7
1 62 103 92 95
1 82 69 23 43
1 41 55 39 82
1 10 66 45 65
1 18 40 26 39
1 100 126 9 112
1 15 7 64 92
1 86 95 21 98
1 98 20 88 125
1 31 79 21 104
1 40 90 39 39
1 27 90 39 114
1 84 67 90 98
1 51 102 73 108
1 69 130 67 35
1 61 98 81 46
1 97 90 8 69
1 51 28 29 65
1 27 54 91 45
1 54 10 11 102
1 87 78 43 117
1 72 59 24 75
1 71 120 37 52
1 46 89 82 82
1 77 67 84 31
1 99 3 61 55
1 91 105 36 111
