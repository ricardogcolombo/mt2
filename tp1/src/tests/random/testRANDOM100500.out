0.518715
0.591997
0.833785
0.638715
0.748098
0.531343
0.379189
0.472300
0.415900
0.578762
0.230691
0.511144
0.398277
0.579059
0.224717
0.297961
0.459045
0.325893
0.542920
0.549948
0.575761
0.865201
0.537529
0.546697
0.470248
0.545468
0.535730
0.608443
0.452048
0.547386
0.425212
0.466105
0.413473
0.479988
0.401621
0.702278
0.307137
0.566909
0.694169
0.552419
0.392200
0.589297
0.650204
0.502588
0.408913
0.321653
0.509877
0.276916
0.548438
0.473095
0.474046
0.325446
0.376483
0.475233
0.421698
0.358051
0.211546
0.273593
0.606232
0.465874
0.729382
0.587177
0.530929
0.426065
0.690815
0.631657
0.672287
0.438863
0.767838
0.553545
0.352255
0.315453
0.289112
0.682169
0.827126
0.365515
0.656953
0.574097
0.288991
0.556847
0.686313
0.446309
0.457119
0.596362
0.554793
0.392402
0.401127
0.620088
0.588802
0.781131
0.349182
0.448035
0.406750
0.275430
0.482756
0.574526
0.570861
0.487415
0.330713
0.429145
